### DISCUSSION TITLE
harvester status through CLI`

### DISCUSSION THREAD

**Discussion by jayhohoho2019** [Chia-Network/chia-blockchain#7159] [Q&A]
I have a few Pi4s running as harvesters only.  I'm wondering how on the full node I would be able to get a complete view of my harvesters, preferably without the GUI, which keeps some info about harvesters in the Farm tab but doesn't seem to keep that updated correctly, along with other limitations.  I'm wondering if there is any CLI that would give me a clean report of all my harvesters (IPs, snapshot info like total plots, etc.).  "chia show" doesn't seem to have that.   Even if I set log level to INFO on the full node, its log file doesn't show all the details of the harvesters, except for the local harvester running on the full node.  Also, is there a limit on the number of harvesters a full node/farmer can be connected to?

**Comment by loppefaaret** [comment]
if you update to 1.2.0 - on everything - it is now able to show remote harvesters in `chia farm summary`

**Reply by jayhohoho2019** [reply]
Great. Thanks for the info. I understand the OG plots may be harvested alongside the new portable plots with v1.2+, is there any special setup to be done for this to happen?  Or it would work by default?    I'm just trying to understand what the best approach would be to upgrade everything to v1.2.0.

**Reply by loppefaaret** [reply]
it just works... there is some display issues, but thats all
